src/api.rs

```diff
@@ -143,6 +154,10 @@ impl EncoderConfig {
       train_rdo: false
     }
   }
+
+  pub fn frame_rate(&self) -> f64 {
```

Comment:
This is often not true, probably we should add a field for that (and consider having a tileinfo builder in the frame invariants and not the other way round. ^^

Reply:
I understand that it will not remain accurate, but isn't it always true *currently*, or can rav1e produce VFR streams, or streams with a time base whose reciprocal is not the frame rate?